pulp_ansible/app/migrations/0030_collectionversion_requires_ansible.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 2.2.17 on 2020-12-09 20:22
+
+import json
+import tarfile
+import yaml
+from django.db import migrations, models
+
+from pulp_ansible.app.tasks.utils import get_file_obj_from_tarball
+
+
+def set_requires_ansible_and_manifest_and_files_json(apps, schema_editor):
```

Comment:
This is pretty slick consolidating the migrations like this for performance reasons. :+1: